randovania/gui/ui_files/preset_generation.ui

```diff
@@ -119,6 +119,53 @@
              </property>
             </widget>
            </item>
+           <item>
+            <widget class="Line" name="line_2_2">
+             <property name="orientation">
+              <enum>Qt::Orientation::Horizontal</enum>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <layout class="QHBoxLayout" name="all_obtainable_layout">
+             <item>
+              <widget class="QLabel" name="all_obtainable_label">
+               <property name="text">
+                <string>[Experimental] Place all items logically:</string>
+               </property>
+              </widget>
+             </item>
+             <item>
+              <widget class="QComboBox" name="all_obtainable_combo">
+               <item>
+                <property name="text">
+                 <string>Disabled</string>
+                </property>
+               </item>
+               <item>
+                <property name="text">
+                 <string>Majors only</string>
+                </property>
+               </item>
+               <item>
+                <property name="text">
+                 <string>All pickups</string>
+                </property>
+               </item>
+              </widget>
+             </item>
+            </layout>
+           </item>
+           <item>
+            <widget class="QLabel" name="all_obtainable_description">
+             <property name="text">
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ensures every (major) item included in the preset is placed logically. This means that regardless of other settings, you will be able to collect every item.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Warning:&lt;/span&gt; Selecting 'All pickups' will make generation fail more frequently.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
we'll probably need to review the descriptions here, but I'd prefer to hash out my name suggestion first. just putting this here so we don't forget to come back to it later

Reply:
something worth considering is changing this text field as each option is selected, as we do for several other settings